schema_cache.rst

```diff
@@ -106,26 +106,43 @@ To refresh the cache without restarting the PostgREST server, send the server pr
      # or in docker-compose
      docker-compose kill -s SIGUSR1 <service>
 
-The above is the manual way to do it. To automate cache reloads, use a database trigger like this:
+Another option is to send a `database notification event <https://www.postgresql.org/docs/current/sql-notify.html>`_ from any PostgreSQL client by executing the ``NOTIFY`` command as follows:
```

Comment:
@laurenceisla I think we need some subheadings for making this more clear. So the first part for reloading would be about "Unix Signals" and the second about "NOTIFY"(postgres native IPC).